static/styles/_components/_side-panel.scss

```diff
@@ -23,24 +23,144 @@
 }
 
 .side-panel {
```

Comment:
This is my fault for the way this was originally implemented, but the glossary also has the `.side-panel` class, so all of these style changes break it. We could either move them to `.side-panel--left` or remove the `.side-panel` class from the glossary. (They started out as similarly functioning things, so made sense for them to share some styles, but they're no pretty different so maybe it makes more sense to break them out).